newcastle_racing_ai/newcastle_racing_ai/controller.py

```diff
@@ -5,22 +5,52 @@
 from ackermann_msgs.msg import AckermannDriveStamped, AckermannDrive
 from .parameters import PARAMETERS
 
+
 class Controller(Node):
+    """A value that means `do nothing`"""
+
+    __null_ads = AckermannDriveStamped(
+        drive=AckermannDrive(
+            steering_angle=0.0,
+            steering_angle_velocity=0.0,
+            speed=0.0,
+            acceleration=0.0,
+            jerk=0.0,
+        )
+    )
```

Comment:
I don't think this is needed. Unless I'm mistaken, when you instantiate an `AckermannDriveStamped()`, all fields are automatically set to zero.